### PR TITLE
chore(wasmlog): Delwasmlog

### PR DESCRIPTION
**Pull request by hunshenshi**
Delete Wasm log when a project is deleted 
https://app.clickup.com/t/866a44nk5

### CODE REVIEW

File: cmd/srv-applet-mgr/apis/wasmlog/del.go
Comment (Liuhaai) [inline cmd/srv-applet-mgr/apis/wasmlog/del.go:1]: It's strange for me to call an api manually to do the job.

Comment (hunshenshi) [reply cmd/srv-applet-mgr/apis/wasmlog/del.go:1]: Yes, it is not normally called. The Wasm log is removed when the project or instance is deleted, this interface is called only in some special cases.

Comment (hunshenshi) [reply cmd/srv-applet-mgr/apis/wasmlog/del.go:1]: Yes, it is not normally called. The Wasm log is removed when the project or instance is deleted, this interface is called only in some special cases.

Comment (hunshenshi) [reply cmd/srv-applet-mgr/apis/wasmlog/del.go:1]: Yes, it is not normally called. The Wasm log is removed when the project or instance is deleted, this interface is called only in some special cases.

File: pkg/modules/wasmlog/wasm_log.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+					se := statusx.FromErr(err)
+					summary = append(summary, &statusx.ErrorField{
+						In:    v.WasmLogID.String(),
+						Field: se.Key,
+						Msg:   se.Desc,
+					})
+				}
+			}
+			if len(summary) > 0 {
+				return status.BatchRemoveAppletFailed.StatusErr().
```
Comment (saitofun) [inline pkg/modules/wasmlog/wasm_log.go:57]: Why remove log return `BatchRemoveAppletFailed` pls define a new status error here.

File: pkg/errors/status/error.go [hunk 2]
```diff
@@ -38,6 +38,8 @@ const (
 	MqttSubscribeFailed
 	// @errTalk MQTT Connect Failed
 	MqttConnectFailed
+	// @errTalk Batch Remove WasmLog Failed
```
Comment (saitofun) [inline pkg/errors/status/error.go:41]: need generation?

**Review by saitofun — APPROVED** [review]

**Pull request merged**